plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/Pom.java

```diff
@@ -254,6 +263,20 @@ void updateDependencies(Collection<Library> selectedLibraries,
     }   
   }
 
+  private static Comment findTestComment(Element dependencies) {
+    NodeList children = dependencies.getChildNodes();
+    
+    for (int i = 0; i < children.getLength(); i++) {
+      Node node = children.item(i);
+      if (node.getNodeType() == Node.COMMENT_NODE) {
+        if (node.getNodeValue().trim().toLowerCase(Locale.US).startsWith("test")) {
```

Comment:
Would be good to say something along the lines that our template includes a `<!— test dependencies —>` to delimit compilation/runtime dependencies from test dependencies.

Reply:
done